### ISSUE TITLE
Japanese Transliteration doesn't make much sense

### ISSUE THREAD

**Issue by BurgerBurglar** [narVidhai/Google-Transliterate-API#4]
``` Python
suggestions = transliterate_word("John", lang_code="ja")
print(suggestions)

# ['女ｈん', '女hん', '助ｈん', '助hん', 'じょhん', 'ジョhン']
```
The results are no where near the original "John".

**Comment by GokulNC** [comment]
Sorry, I do not know Japanese. Can you please check if it's working properly here:
https://www.google.com/intl/ja/inputtools/try/

**Comment by BurgerBurglar** [comment]
It works exactly like the options. So I may have got the intention of this repo wrong, it doesn't transliterate names "America" directly to another language, but rather tries to write "A M E R I C A" in another language, and see what it can get?

**Comment by GokulNC** [comment]
OK. I am not sure. I just quickly went through how Japanese writing system works.

There are 2 possibilities to write "John":
- Katakana: ジョン
- Hiragana: ぢょん

(I suppose there's no kanji form for those non-native words)

So which kana do you expect as output?

---

BTW, this library is just a wrapper to the Google's Transliteration API. So the transliteration logic is not designed by us :)

**Comment by BurgerBurglar** [comment]
> transliteration logic is not designed by us 

Yeah I get that now. The universally accepted way to write John is the former.

Thanks though, great tool :)
